engine/src/main/scala/cromwell/engine/workflow/WorkflowDockerLookupActor.scala

```diff
@@ -249,7 +251,7 @@ object WorkflowDockerLookupActor {
   final case class WorkflowDockerTerminalFailure(reason: Throwable, request: DockerHashRequest) extends WorkflowDockerLookupResponse
 
   def props(workflowId: WorkflowId, dockerHashingActor: ActorRef, startMode: StartMode, databaseInterface: SqlDatabase = SingletonServicesStore.databaseInterface) = {
-    Props(new WorkflowDockerLookupActor(workflowId, dockerHashingActor, startMode, databaseInterface))
+    Props(new WorkflowDockerLookupActor(workflowId, dockerHashingActor, startMode, databaseInterface)).withDispatcher(EngineDispatcher)
```

Comment:
Questions because I don't really understand all this stuff:
- Does the engine dispatcher also handle all of the mission-critical EJEAs, JABJEAs, WMAs, etc?
- Are there benefits of having a `DbDispatcher` for DB stuff?